docs/pages/enroll-resources/machine-id/deployment/kubernetes.mdx

```diff
@@ -124,6 +124,11 @@ $ kubectl apply -f ./k8s-rbac.yaml
 
 ## Step 3/5. Create a join token
 
+<Admonition type="note">
+If you are using AWS EKS clusters with IAM OIDC enabled, you will need to use [`in_cluster`](../../../reference/join-methods.mdx#kubernetes-in-cluster) 
```

Comment:
I think we should be advising people to use the `iam` join method rather than `in_cluster` - `in_cluster` only works where the Teleport Auth Server is installed in the same EKS cluster,

Reply:
ack! please lmk if the change looks ok

Reply:
Ah - so the problem is that guide is for agent joining, which is a little different to the bot joining. Hypothetically, if someone merged both this guide and that guide, they'd get to the right-ish thing here. However, at the moment, we don't really have an AWS EKS + Machine ID using IAM joining guide. I know customers have figured it out in the past, but just linking to the EKS IAM guide for agents will likely confuse people more I think.

Reply:
Hi Noah, thanks for the clarification! I will revisit this at another time.